src/encoded/static/components/dataset.js

```diff
@@ -2036,7 +2036,12 @@ export const SeriesComponent = ({ context, title, tableColumns, breadcrumbs, opt
     // the code takes this into account
     const diseases = [
         ...new Set(context.related_datasets.reduce((accumulatedDiseases, currentRelatedDataset) => {
-            const diseasesTermNames = currentRelatedDataset.replicates.map((replicate) => replicate.library.biosample.disease_term_name);
+            const diseasesTermNames = currentRelatedDataset.replicates.map((replicate) => {
+                if (typeof replicate !== 'string') {
```

Comment:
Let's talk about this. Why are you checking for type rather than property the object has or states like falsey